client/landing/subscriptions/components/add-sites-modal/styles.scss

```diff
@@ -12,5 +12,35 @@
 		font-style: normal;
 		font-weight: 400;
 		line-height: 20px;
+		margin-bottom: 1em;
+	}
+
+	.reader-feed-item__site-preview-h-stack {
+		max-width: 200px;
```

Comment:
Reducing the width of site avatar on modal to make sure we have good space for description.